cmsmon-tools.spec

```diff
@@ -47,10 +57,18 @@ popd
 
 %install
 cd ../%pkg-%ver
-cp src/go/MONIT/monit %i/
+for cmd in %monit_commands; do
+  cp src/go/MONIT/$cmd %i/
+done
 for cmd in %cmsmon_commands; do
   cp src/go/NATS/$cmd %i/
 done
+# add prometheus, alertmanager tools to our install area
+cd ../prometheus-%promv.%arch
+cp promtool %i/
+cp prometheus %i/
+cd ../alertmanager-%amver.%arch
+cp amtool %i/
```

Comment:
@vkuznet , do you need these new tools in the $RPM_INSTALL_PREFIX/cmsmon ?  If yes then please add `promtool prometheus  amtool` also in the `for` loop at the end.

Reply:
good point, I added them